runtimes/datascience/ubi9-python-3.11/Pipfile

```diff
@@ -7,41 +7,43 @@ name = "pypi"
 
 [packages]
 # Datascience and useful extensions
-boto3 = "~=1.34.50"
-kafka-python = "~=2.0.2"
-matplotlib = "~=3.8.3"
-numpy = "~=1.26.4"
-pandas = "~=2.2.0"
-plotly = "~=5.20.0"
-scikit-learn = "~=1.4.0"
-scipy = "~=1.12.0"
-skl2onnx = "~=1.16.0"
-codeflare-sdk = "~=0.19.1"
-memray = "~=1.10.0"  # any version above this would cause issue with rich package
+boto3 = "~=1.35.14"
+kafka-python-ng = "~=2.2.2"
+matplotlib = "~=3.9.2"
+numpy = "~=2.1.0"
+pandas = "~=2.2.2"
+plotly = "~=5.23.0"
+scikit-learn = "~=1.5.1"
+scipy = "~=1.14.1"
+skl2onnx = "~=1.17.0"
+onnxconverter-common = "~=1.13.0" # Required for skl2onnx, as upgraded version is not compatible with protobuf
+codeflare-sdk = "~=0.18.0"
```

Comment:
Is this version decrease intentional? :thinking:  (same applies for this change in the rest of the files)

Reply:
Yeah, because on jupyter ds notebook we have `onnxconverter-common = "~=1.13.0"`  I kept the same on the runtime as well

Reply:
I understand the reason for the onnxconverter-common - but my question was for the codeflare-sdk.

Reply:
I looked [here](https://github.com/opendatahub-io/notebooks/blob/main/jupyter/pytorch/ubi9-python-3.11/Pipfile#L31) - so I guess you tried to aligned these images... well, it's probably not a big deal for now as the regular codeflare-sdk update should fix it for all the images. But we should be sure we'll run it explicitly before the release then...